### PR TITLE
Add bulk sending guidance page

### PR DESCRIPTION
**Pull request by saimaghafoor**
This PR adds a new guidance page explaining bulk sending. It also adds links to the new page to the guidance landing page and left-hand navigation.

The page includes information about daily sending limits – so we shouldn’t merge it until we’re ready to communicate that change to our users.

### CODE REVIEW

**Comment by idavidmcdonald** [comment]
@saimaghafoor, currently the page that shows this content is available at https://www.notifications.service.gov.uk/using-notify/guidance/create-and-send-messages. Are you saying you would like the url changed to something (potentially https://www.notifications.service.gov.uk/using-notify/guidance/send-messages-in-bulk ?)

**Comment by saimaghafoor** [comment]
Yes please @idavidmcdonald :)

**Comment by idavidmcdonald** [comment]
So how we do URL changes is we always want the current URL to continue working (even if it redirects to the new location).

Are you happy for the current URL to redirect to the new URL?

**Comment by karlchillmaid** [comment]
@idavidmcdonald might be worth holding off on doing anything with this one for the time being.

@samuelhwilliams and I are working to reconcile a couple of other PRs that need to be merged before this one.

**Comment by idavidmcdonald** [comment]
I might hand this one over to the three of you to resolve if that is OK? As it seems fairly related to features work and it looks like you'll be best placed to give it to the proper attention

File: app/templates/views/guidance/create-and-send-messages.html [hunk 1]
```diff
@@ -6,21 +6,20 @@
...
+  <h1 class="heading-large">Send messages in bulk</h1>
 
-  <p class="govuk-body"><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.register') }}">Create an account</a> to see a short tutorial explaining how to create and send messages.</p>
+  <p class="govuk-body">You can send a batch of up to 100,000 messages at once.</p>
+<p class="govuk-body>"There's a maximum daily limit of 250,000 emails, 250,000 text messages and 25,000 letters. If you need to discuss these limits, <a href="https://www.notifications.service.gov.uk/support">contact us</a>.</p>
 
-  <h2 class="heading-medium" id="bulk-sending">Bulk sending</h2>
-
-  <p class="govuk-body">To send a batch of messages at once:</p>
+  <p class="govuk-body">To send a batch of messages at:</p>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/create-and-send-messages.html:19]:
```suggestion
  <p class="govuk-body">To send a batch of messages:</p>
```

File: app/templates/views/guidance/bulk-sending.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  Bulk sending
+{% endblock %}
+
+{% block content_column_content %}
+
+  <h1 class="heading-large">Bulk sending</h1>
+
+  <p class="govuk-body">You can send a batch of up to 100,000 messages at once.</p>
+
+  <p class="govuk-body>"There's a maximum daily limit of 250,000 emails, 250,000 text messages and 25,000 letters. If you need to discuss these limits, <a href="https://www.notifications.service.gov.uk/support">contact us</a>.</p>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/bulk-sending.html:18]:
```suggestion
  <p class="govuk-body>There's a maximum daily limit of 250,000 emails, 250,000 text messages and 25,000 letters. If you need to discuss these limits, <a href="https://www.notifications.service.gov.uk/support">contact us</a>.</p>
```

File: app/templates/views/guidance/bulk-sending.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  <p class="govuk-body">To send a batch of messages at:</p>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/bulk-sending.html:20]:
```suggestion
  <p class="govuk-body">To send a batch of messages:</p>
```

File: app/templates/views/guidance/bulk-sending.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  
+  <ol class="govuk-list govuk-list--number">
+    <li>Go to the {{ service_link(current_service, 'main.choose_template', 'templates') }}.</li>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/bulk-sending.html:23]:
```suggestion
    <li>Go to the {{ service_link(current_service, 'main.choose_template', 'templates') }} page.</li>
```

File: app/templates/views/guidance/bulk-sending.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    <li>Add a new template or choose an existing template and select <b class="govuk-!-font-weight-bold">Get ready to send</b>.</li>
+    <li>Select <b class="govuk-!-font-weight-bold">Upload a list of email addresses</b>, <b class="govuk-!-font-weight-bold">Upload a list of phone numbers</b>, or <b class="govuk-!-font-weight-bold">Upload a list of addresses</b>.</li>
+  </ol>
+
+  <p class="govuk-body">You can also choose when to send your messages. Find out <a href="https://www.notifications.service.gov.uk/using-notify/guidance/schedule-emails-and-text-messages">more about scheduling messages</a>.</p>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/bulk-sending.html:28]:
```suggestion
  <p class="govuk-body">You can also <a href="https://www.notifications.service.gov.uk/using-notify/guidance/schedule-emails-and-text-messages">schedule a batch of emails and text messages</a>.</p>
```

File: app/templates/views/guidance/bulk-sending.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  Bulk sending
+{% endblock %}
+
+{% block content_column_content %}
+
+  <h1 class="heading-large">Bulk sending</h1>
+
+  <p class="govuk-body">You can send a batch of up to 100,000 messages at once.</p>
+
+  <p class="govuk-body">There’s a maximum daily limit of 250,000 emails, 250,000 text messages and 25,000 letters. If you need to discuss these limits, <a href="https://www.notifications.service.gov.uk/support">contact us</a>.</p>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/bulk-sending.html:18]:
```suggestion
  <p class="govuk-body">There’s a maximum daily limit of 250,000 emails, 250,000 text messages and 20,000 letters. If you need to discuss these limits, <a href="https://www.notifications.service.gov.uk/support">contact us</a>.</p>
```

**Review by samuelhwilliams — APPROVED** [review]

File: app/templates/views/guidance/bulk-sending.html [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  Bulk sending
+{% endblock %}
+
+{% block content_column_content %}
+
+  <h1 class="heading-large">Bulk sending</h1>
+
+  <p class="govuk-body">You can send a batch of up to 100,000 messages at once.</p>
+
+  <p class="govuk-body">There’s a maximum daily limit of 250,000 emails, 250,000 text messages and 20,000 letters. If you need to discuss these limits, <a href="https://www.notifications.service.gov.uk/support">contact us</a>.</p>
```
Comment (samuelhwilliams) [inline app/templates/views/guidance/bulk-sending.html:18]: We aren't enforcing this limit yet - just something to be aware of. If we merge this before we are, we might confuse some people

Comment (karlchillmaid) [reply app/templates/views/guidance/bulk-sending.html:18]:
```suggestion
  <!-- <p class="govuk-body">There’s a maximum daily limit of 250,000 emails, 250,000 text messages and 20,000 letters. If you need to discuss these limits, <a href="https://www.notifications.service.gov.uk/support">contact us</a>.</p> -->
```

**Pull request merged**